plugins/polio/js/src/domains/Chronogram/Chronogram/api/useGetChronogram.ts

```diff
@@ -9,7 +9,11 @@ import { apiBaseUrl } from '../../constants';
 const getChronogram = async (
     params: Partial<ChronogramParams>,
 ): Promise<ChronogramApiResponse> => {
-    const queryString = new URLSearchParams(params).toString();
+    const { country, ...paramsWithoutCountry } = params;
+    let queryString = new URLSearchParams(paramsWithoutCountry).toString();
+    if (country) {
+        queryString += `&country=${country.split(',').join('&country=')}`;
```

Comment:
I see that you used the other format for multiple parameters, it's interesting. 

What happens in the backend if you need to process these parameters/convert them to something else? Do you do that somewhere in the `ChronogramFilter` class?

Reply:
> I see that you used the other format for multiple parameters, it's interesting.

Yes, but this is a bit hackish 🤫

> What happens in the backend if you need to process these parameters/convert them to something else? Do you do that somewhere in the `ChronogramFilter` class?

I guess you can. But usually these parameters are mapped to a Django choice and you don't have anything to do.